### PR TITLE
link to the cookbook article on avoiding to start a session

### PR DESCRIPTION
**Pull request by dbu**
resolve the todo now that #4661 is merged

### CODE REVIEW

File: cookbook/cache/varnish.rst [hunk 1]
```diff
@@ -72,11 +72,10 @@ If you know for sure that the backend never uses sessions or basic
 authentication, have varnish remove the corresponding header from requests to
 prevent clients from bypassing the cache. In practice, you will need sessions
 at least for some parts of the site, e.g. when using forms with
-:ref:`CSRF Protection <forms-csrf>`. In this situation, make sure to only
-start a session when actually needed, and clear the session when it is no
-longer needed. Alternatively, you can look into :doc:`../cache/form_csrf_caching`.
-
-.. todo link "only start a session when actually needed" to cookbook/session/avoid_session_start once https://github.com/symfony/symfony-docs/pull/4661 is merged
+:ref:`CSRF Protection <forms-csrf>`. In this situation, make sure to
+:doc:`only start a session when actually needed <../session/avoid_session_start>`,
```
Comment (xabbuh) [inline cookbook/cache/varnish.rst]: Please use an absolute path here (`/cookbook/session/avoid_session_start`).

Comment (xabbuh) [inline cookbook/cache/varnish.rst]: And please also remove the serial comma.

File: cookbook/cache/varnish.rst [hunk 2]
```diff
@@ -72,11 +72,10 @@ If you know for sure that the backend never uses sessions or basic
...
+and clear the session when it is no longer needed. Alternatively, you can look
+into :doc:`../cache/form_csrf_caching`.
```
Comment (xabbuh) [inline cookbook/cache/varnish.rst]: same here

**Comment by dbu** [comment]
fixed the issues @xabbuh

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
I love interlinking. Thank you David!

**Pull request merged**